gensim/corpora/dictionary.py

```diff
@@ -463,7 +458,7 @@ def save_as_text(self, fname, sort_by_word=True):
         fname : str
             Path to output file.
         sort_by_word : bool, optional
-            Sort words in lexicographical order before writing them out?
+            Sort words in lexicographical order before writing them out.
```

Comment:
-1: bool should be a question, to make it clear what positive answer means (True).

Reply:
Fixed